config/read.go

```diff
@@ -55,6 +55,11 @@ func getDefaultConfig() *ServerConfig {
 	if apiKey := os.Getenv("PGA_API_KEY"); apiKey != "" {
 		config.APIKey = apiKey
 	}
+	// Since there's a discrepancy here between the config file key (api_base_url) and the
+	// env var key (PGA_API_BASEURL), also accept PGA_API_BASE_URL to make our lives easier.
```

Comment:
Alternately, we could standardize on this one for consistency (and still accept the old spelling).

Reply:
Assuming you mean `baseurl` with this one (i.e. without the `_`), I think that will be a bit more annoying to handle for the ini field names, so it seems better to deprecate `PGA_API_BASEURL` and use `BASE_URL` going forward.

Reply:
No, I meant align with the config value (with the dash). So I'll go ahead and make the new spelling canonical (and update docs).